content/_index.md

```diff
@@ -14,6 +14,11 @@ carousel:
     ref: covidcast
     alt: Explore COVIDcast
     image: covidcast_withfill.jpg
+  - pre: Our Tools
+    title: Forecast Evaluation Dashboard
+    ref: forecast-eval
```

Comment:
@sgratzl is there somewhere I need to define "forecast-eval" ref so that it points to {currentUrl}/forecast-eval?

Reply:
please use the link version instead, e.g., `link: /forecast-eval` refs are internal hugo references.

Reply:
Thanks! Ready for review now